src/main/java/io/aiven/kafka/auth/json/AivenAcl.java

```diff
@@ -38,26 +38,34 @@ public class AivenAcl {
     @SerializedName("resource_pattern")
     public final String resourceRePattern;
 
+    @SerializedName("permission_type")
+    private final AclPermissionType permissionType;
+
```

Comment:
This class is getting more complex and there are some assumptions that would be nice to check explicitly. For example, that the JSON library converts `ALLOW` and `DENY` correctly into the enum. I.e. we need some tests for `AclJsonReaderTest` (maybe as a separate PR).

Reply:
I added `AclJsonReaderTest`